x-pack/journalbeat/magefile.go

```diff
@@ -0,0 +1,83 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
I've no clue about the mage build process for journalbeat, so I copied the file from heartbeats, but it might not work, who can help with this?

Reply:
AFAIK we build everything from `journalbeat` ATM. Why is this PR necessary? We there is nothing in the X-Pack version of Journalbeat.

Reply:
Aha, thanks for the clarification, if that's the case, then I'll close this particular, PR I thought `x-pack/journalbeat` was missing from the CI